### DISCUSSION TITLE
Párhuzamos access tokenek

### DISCUSSION THREAD

**Discussion by rrd108** [nav-gov-hu/M2M#50] [Kérdések] [answered]
Engedi a rendszer, hogy egy usernek egyszerre két érvényes access tokenje legyen, vagy a `createToken` hívás érvényteleníti a korábbi tokeneket?

Pl ha az `addFile`, a `createDocument` vagy  a `validateDocument` 30 mp-nél tovább tart, akkor az állapotot később újra le kell kérdezni.

Mivel ezt egy külön folyamat fogja elvégezni felmerült a kérdés, hogy lehet-e egyszerre két érvényes access token.

**Comment by TamasPill** [comment, marked as answer]
Minden kiállított token 10 percig érvényes, ezen időablakon belül akár több tokennel is rendelkezhet a felhasználó.
Ha a getDocument lekérdezés másik tokennel történik, az nem jelent problémát.

**Reply by rrd108** [reply]
szuper, ez egyszerűsíti a folyamatot.

köszi a választ
